src/map/clif.c

```diff
@@ -4744,6 +4746,23 @@ static void clif_storageclose(struct map_session_data *sd)
 	WFIFOSET(fd,packet_len(0xf8));
 }
 
+/// Notifies clients in an area of a player's souls.
+/// 01d0 <id>.L <amount>.W (ZC_SPIRITS)
+/// 01e1 <id>.L <amount>.W (ZC_SPIRITS2)
+/// 0b73 <id>.L <amount>.W
+void clif_soulball(struct map_session_data *sd, struct block_list *bl, enum send_target target)
+{
+	unsigned char buf[16];
+
+	nullpo_retv(sd);
+
+	WBUFW(buf, 0) = 0x1d0;
```

Comment:
I'm thinking if I have to drop `clif_spiritball` into more of a `clif_displayball` and merge both `clif_soulball` and `clif_spiritball` since both are actually doing the same. I guess i'll use the parameter list of `clif_soulball` rather `clif_spiritball` to make it more flexible. Though i'm not sure if i should include it in this PR. Maybe i can do it in another PR?